distribution/launcher/README.md

```diff
@@ -0,0 +1,24 @@
+# Enso CLI
+
+> Actionables for this file:
+>
+> - This is just a draft of the file that will be included inside the launcher
+>   distribution. It should be completed when the launcher is implemented.
+
+## Installation
+The Enso distribution is portable, you can put this directory wherever you want
+on your system. For example you can put it at `~/.enso`.
+
+It is a good idea to add the `bin` directory to your system `PATH` so that you
+can run the launcher just by typing `enso`.
```

Comment:
I'm not sure we need to make the distinction that what people run is a `launcher`. Maybe just "so that you can run Enso...".